pom.xml

```diff
@@ -1760,6 +1767,19 @@
             </execution>
           </executions>
         </plugin>
+
+        <plugin>
+          <groupId>io.github.zlika</groupId>
+          <artifactId>reproducible-build-maven-plugin</artifactId>
+          <version>${reproducible-build-maven-plugin.version}</version>
+          <executions>
+            <execution>
+              <goals>
+                <goal>strip-jar</goal>
```

Comment:
Well the thing is that java adds timestamps in some of the jar elements, e.g. the MANIFEST file and in some cases in the files that are updated with last modification date. The plugin covers these and other cases, you can take a look at the presentation he did (is really short) for some more detail. https://zlika.github.io/presentations/devoxx_fr_2016/reproducible-builds/slides_en.html#/_pom_properties